package.json

```diff
@@ -73,5 +73,8 @@
   },
   "scripts": {
     "postinstall": "chmod u+x .postinstall.js && node ./.postinstall.js"
+  },
+  "frontendDependenciesCSS": {
```

Comment:
mhmm. otherwise `frontendDependenciesCSS` connects more obvious to the `dependencyConfiguration`  thingy …